.devops/repo/src/commands/build.rs

```diff
@@ -0,0 +1,31 @@
+use crate::utils::{execute, print_applications};
+use anyhow::anyhow;
+use std::process::Command;
+
+const MESSAGE: &str = "Errors found in";
+
+pub(crate) fn run(apps: Vec<String>, root: String) {
+    let mut errors: Vec<String> = vec![];
+
+    for app in apps {
+        println!("\nBuilding {}", app);
+
+        let project = std::fs::canonicalize(format!("{}/{}/project", root, app))
+            .map_err(|error| {
+                anyhow!(
+                    "Failed to canonicalize path to project for app '{}': {}",
+                    app,
+                    error
+                )
+            })
+            .unwrap();
```

Comment:
```suggestion
```

Ah, to make full use of `anyhow` and avoid `unwrap()`, we probably want to use `?` here and make `run()` return an `anyhow::Result`.